executor/insert_common.go

```diff
@@ -365,7 +364,7 @@ func (e *InsertValues) doBatchInsert(ctx context.Context) error {
 	}
 	if err := e.ctx.NewTxn(ctx); err != nil {
 		// We should return a special error for batch insert.
-		return ErrBatchInsertFail.GenWithStack("BatchInsert failed with error: %v", err)
+		return ErrBatchDMLFail.GenWithStack("BatchInsert failed with error: %v", err)
```

Comment:
where does `doBatchInsert` will be called?

Reply:
It should be useless. I'll remove it.

Reply:
addressed, PTAL